volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -108,7 +108,7 @@ def get_symbols(self):
                                                                  "linux",
                                                                  "elf",
                                                                  native_types = None,
-                                                                 class_types = extensions.elf.class_types)
+                                                                 class_types = elf.class_types)
```

Comment:
Good catch, but since this is a critical and standalone fix, I've made it directly to the tree.  It just makes it decouples the pull request from a problem fix, in case we need to track it down at some point.